### DISCUSSION TITLE
Disable @ in URLs

### DISCUSSION THREAD

**Discussion by PixelsCommander** [imgproxy/imgproxy#829] [Q&A]
Some of our files have @ in names and the whole proxying setup is not really programmable. Wondering if disabling extensions @ is possible via configuration? Because I d expect some situations where this feature hurts but brings no benefits.

**Comment by DarthSim** [comment]
You need to URL-encode `@` in your source URLs. Disabling extensions in the imgproxy URLs is currently not possible.
